Shared/Views/NotificationSettings.swift

```diff
@@ -67,56 +68,59 @@ class NotificationSettingsData: ObservableObject {
-    
+        
     var body: some View {
-        Form {
-            Toggle(isOn: self.$data.showMilestones, label: {
-                Text("Milestones")
-            })
-                .disabled(!self.data.notificationsAllowed)
-            Toggle(isOn: self.$data.showGoal, label: {
-                Text("Goal Reached")
-            })
-                .disabled(!self.data.notificationsAllowed)
-            Toggle(isOn: self.$data.showSignificantAmounts, label: {
-                Text("Significant Amounts")
-            })
-                .disabled(!self.data.notificationsAllowed)
-            Toggle(isOn: self.$data.showMilestoneAdded, label: {
-                Text("Milestones Added")
-            })
-                .disabled(!self.data.notificationsAllowed)
-            if(!self.data.notificationsAllowed) {
-                Section() {
-                    Button(action: {
-                        if let url = URL(string: UIApplication.openSettingsURLString) {
-                            if(UIApplication.shared.canOpenURL(url)) {
-                                UIApplication.shared.open(url, options: [:], completionHandler: {_ in})
-                            }
-                        }
-                    }, label: {
-                        Text("Notification access denied.")
+        NavigationView {
+            Section(header: EmptyView().accessibility(hidden: true)) {
+                Form {
+                    Toggle(isOn: self.$data.showMilestones, label: {
+                        Text("Milestones")
                     })
+                        .disabled(!self.data.notificationsAllowed)
+                    Toggle(isOn: self.$data.showGoal, label: {
+                        Text("Goal Reached")
+                    })
+                        .disabled(!self.data.notificationsAllowed)
+                    Toggle(isOn: self.$data.showSignificantAmounts, label: {
+                        Text("Significant Amounts")
+                    })
+                        .disabled(!self.data.notificationsAllowed)
+                    Toggle(isOn: self.$data.showMilestoneAdded, label: {
+                        Text("Milestones Added")
+                    })
+                        .disabled(!self.data.notificationsAllowed)
+                    if(!self.data.notificationsAllowed) {
+                        Section() {
+                            Button(action: {
+                                if let url = URL(string: UIApplication.openSettingsURLString) {
+                                    if(UIApplication.shared.canOpenURL(url)) {
+                                        UIApplication.shared.open(url, options: [:], completionHandler: {_ in})
+                                    }
+                                }
+                            }, label: {
+                                Text("Notification access denied.")
+                            })
+                        }
+                    }
```

Comment:
Not quite the right nesting here - `Section` goes inside the `Form`, and we want the other `Section` (line 96, here) to be outside the first one, and also have the 'no header' workaround applied. (I actually tried to do this myself, but GitHub is doing Permission Issues at me 🤷‍♂️)

Reply:
i.e.:
```swift
Form {
    Section(header: EmptyView().accessibility(hidden: true)) {
        Toggle(isOn: self.$data.showMilestones, label: {
            Text("Milestones")
        })
            .disabled(!self.data.notificationsAllowed)
        Toggle(isOn: self.$data.showGoal, label: {
            Text("Goal Reached")
        })
            .disabled(!self.data.notificationsAllowed)
        Toggle(isOn: self.$data.showSignificantAmounts, label: {
            Text("Significant Amounts")
        })
            .disabled(!self.data.notificationsAllowed)
        Toggle(isOn: self.$data.showMilestoneAdded, label: {
            Text("Milestones Added")
        })
            .disabled(!self.data.notificationsAllowed)
        
    }
    if(!self.data.notificationsAllowed) {
        Section(header: EmptyView().accessibility(hidden: true)) {
            Button(action: {
                if let url = URL(string: UIApplication.openSettingsURLString) {
                    if(UIApplication.shared.canOpenURL(url)) {
                        UIApplication.shared.open(url, options: [:], completionHandler: {_ in})
                    }
                }
            }, label: {
                Text("Notification access denied.")
            })
        }
    }
}
```